pkg/utils/time/time.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package time contains time specific functions and variable to make it to overwrite for unit tests
+package time
+
+import "time"
+
+var Now = time.Now
```

Comment:
I don't see this used anywhere.
Also, I don't understand how it would be used...  What is the different in calling `time.Now()` from go and `time.Now()` from this new package?

Reply:
I will remove this now. I used in the earlier version of this PR for unit tests but I removed that because we removed the expiration time from the log message.

Reply:
I will keep this for now and update the https://github.com/vmware-tanzu/tanzu-cli/pull/819#discussion_r1790783947 as part of the rebase.